### PR TITLE
task: Update release notes for v24.10

### PR DESCRIPTION
**Pull request by e0ne**

### CODE REVIEW

**Review by rollandf — APPROVED** [review]

File: docs/release-notes.rst [hunk 1]
```diff
@@ -36,6 +36,9 @@ Changes and New Features
 
    * - Version
      - Description
+   * - 24.10.0
+     - | - Added support for NVIDIA NIC Configuration Operator deployment.
+       | - Added support for Support Single-Node OpenShift (SNO).
```
Comment (rollandf) [inline docs/release-notes.rst:41]: How do we communicate the change regarding the NCP not being created by helm anymore?

Comment (e0ne) [reply docs/release-notes.rst:41]: done

File: docs/release-notes.rst [hunk 2]
```diff
@@ -36,6 +36,10 @@ Changes and New Features
...
+       | - NVIDIA Network Operator Helm chart does not deploy NicClusterPolicy CRD anymore.
```
Comment (rollandf) [inline docs/release-notes.rst:42]: WDYT
- NVIDIA Network Operator Helm chart does not create NicClusterPolicy CR anymore.

**Review by rollandf — CHANGES_REQUESTED** [review]
Please consider:
NVIDIA Network Operator Helm chart does not create NicClusterPolicy CR anymore.

File: docs/release-notes.rst (hunk 2 above)
Comment (e0ne) [reply docs/release-notes.rst:42]: I like this proposal!

**Review by rollandf — APPROVED** [review]

**Pull request merged**